client/packages/system/src/Encounter/ListView/columns.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  useColumns,
+  ColumnAlign,
+  Column,
+  ColumnDescription,
+  SortBy,
+} from '@openmsupply-client/common';
+import { useFormatDateTime } from '@common/intl';
+import { EncounterRowFragment } from '../api';
+import { ProgramEventFragment } from '../api/operations.generated';
+
+const encounterEventCellValue = (events: ProgramEventFragment[]) => {
+  // just take the name of the first event
+  return events[0]?.name ?? '';
+};
+
+interface useEncounterListColumnsProps {
+  onChangeSortBy: (column: Column<any>) => void;
+  sortBy: SortBy<EncounterRowFragment>;
+  includePatient?: boolean;
+}
+
+export const useEncounterListColumns = ({
+  onChangeSortBy,
+  sortBy,
+  includePatient = false,
+}: useEncounterListColumnsProps) => {
+  const { localisedDate, localisedTime } = useFormatDateTime();
+  includePatient;
+
+  const columnList: ColumnDescription<EncounterRowFragment>[] = [
+    {
+      key: 'encounter-type',
+      label: 'label.encounter-type',
+      sortable: false,
+      accessor: ({ rowData }) => rowData?.document.documentRegistry?.name,
+    },
+    {
+      key: 'program',
+      label: 'label.program',
+    },
+    {
+      key: 'startDatetime',
+      label: 'label.date',
+      accessor: ({ rowData }) => rowData?.startDatetime,
+      formatter: dateString =>
+        dateString ? localisedDate((dateString as string) || '') : '',
+    },
+    {
+      key: 'startTime',
+      label: 'label.encounter-start',
+      sortable: false,
+      accessor: ({ rowData }) => rowData?.startDatetime,
```

Comment:
oops. this was missed earlier, and there is no `startTime` column so no data was shown